### ISSUE TITLE
Updating R version

### ISSUE THREAD

**Issue by kegro** [GlobWetlandAfrica/gwautils#4]
@redfrexx we are making a new installation for QGIS 3. Do you see any issue with us wanting to update the R version? It seems this package was build under R 3.3.2. I'm not sure if we mess with the R version will it affect this package or dependencies or any of the workflows. What do you think?
PS: hope all is going well with your PhD!

**Comment by redfrexx** [comment]
Hey Kenneth! Glad to hear you're updating the toolbox! I haven't used R scripts within QGIS 3 yet, but it seems like QGIS 3 doesn't support the execution of R scripts inside the processing tool box anymore (see [here](https://gis.stackexchange.com/questions/273623/r-scripts-in-qgis-3-0)). Seems like the altervative is using the [Processing R Provider plugin](https://plugins.qgis.org/plugins/processing_r/). Would it be possible to use this plugin to run the R scripts inside the GWA toolbox workflow? 

PS: Thanks, yes it's going well, just needs a lot of patience. ;) Hope you're doing fine as well!

**Comment by kegro** [comment]
Thanks for the quick reply Christina! Yes all going pretty well here thanks :) 
We've tried with the Processing R provider as you mention, and it seems to work. 
My worry was more that some of the packages we use might not work or throw an unexpected error if we chose a later version of R. For example the GWAutils you made. Would this work in a later version of R? I can't see if it uses any other packages as its in the compiled form. and what about the other worflows you think?

**Comment by kegro** [comment]
@redfrexx following from above. . . I'm wondering does GWAutils load any other packages, or do the packages you use get loaded in the workflow e.g. https://github.com/GlobWetlandAfrica/GWA_scripts/blob/a6aa5ffacf0077cbe74e5becf6a510807312d69e/rscripts/GWA_TBX/05_WCR_OPT_WaterDetection.rsx#L36

**Comment by redfrexx** [comment]
Yes, I think these are all the packages. I can't say for sure if it will work with a later version, would have to be tested. Especially if you plan to use R version 4. The other workflows are all in Python 2 as far as I remember. So in QGIS 3 you would have to adapt them to Python 3. But that should generally be possible, just requires some time to adapt and test. I think Björn from Geoville was the last one to work with the scripts. Maybe he can help with that?
